code/modules/cm_preds/yaut_bracers.dm

```diff
@@ -643,12 +648,17 @@
 	set category = "Yautja.Misc"
 	set src in usr
 
+	if(explosion_type == SD_TYPE_BIG && exploding)
+		to_chat(usr, SPAN_WARNING("Why would you want to do this?"))
+		return
+
 	if(alert("Which explosion type do you want?","Explosive Bracers", "Small", "Big") == "Big")
 		explosion_type = 0
 		log_attack("[key_name_admin(usr)] has changed their Self Destruct to Large")
 	else
 		explosion_type = 1
```

Comment:
lampost when I make you add in defines you know you have to add it for the entire thing right?